Tests/Benchmarks/Benchmarks.csproj

```diff
@@ -15,5 +15,8 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Data.DataSetExtensions" Condition="'$(TargetFramework)' == 'net472'" />
+    <Reference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net472'">
+      <HintPath>..\..\..\..\Program Files\dotnet\sdk\7.0.102\Microsoft\Microsoft.NET.Build.Extensions\net461\lib\System.Net.Http.dll</HintPath>
```

Comment:
This hintpath doesn't look ready for any machine where the repo was cloned at an arbitrary location.